evm/forge-test/TokenBridgeRelayer.t.sol

```diff
@@ -1518,65 +1628,25 @@ contract TokenBridgeRelayerTest is Helpers, ForgeHelpers, Test {
             toNativeTokenAmount,
             normAmounts.tokenDecimals
         );
-
-        // test setup
-        {
-            // target contract setup
-            altRelayer.registerToken(altRelayer.chainId(), wrappedAsset);
-
-            // register this contract as the foreign emitter
-            altRelayer.registerContract(
+        normAmounts.relayerFee = normalizeAmount(
+            altRelayer.calculateRelayerFee(
                 ethereumChainId,
-                addressToBytes32(address(this))
-            );
-
-            // set the native swap rate
-            altRelayer.updateSwapRate(
-                altRelayer.chainId(),
                 wrappedAsset,
-                6.9e2 * altRelayer.swapRatePrecision() // swap rate
-            );
-
-            // set the max to native amount
-            altRelayer.updateMaxNativeSwapAmount(
-                altRelayer.chainId(),
-                wrappedAsset,
-                1e12 // max native swap amount
-            );
-
-            /**
-             * NOTE: The relayer fee is calculated on the source chain, and the
-             * target relayer contract will pay the relayer the encoded value. We
-             * need to simulate calculating the value based on information stored
-             * in the target chain.
-             */
-
-            // set relayer fee to 20 USD
-            altRelayer.updateRelayerFee(
-                ethereumChainId,
-                20 * altRelayer.relayerFeePrecision()
-            );
-
-            normAmounts.relayerFee = normalizeAmount(
-                altRelayer.calculateRelayerFee(
-                    ethereumChainId,
-                    wrappedAsset,
-                    normAmounts.tokenDecimals
-                ),
                 normAmounts.tokenDecimals
-            );
+            ),
+            normAmounts.tokenDecimals
+        );
 
-            // make some assumptions about the fuzz test values
-            vm.assume(
-                normAmounts.transferAmount > 0 &&
-                amount < type(uint96).max
-            );
-            vm.assume(
-                normAmounts.toNative > 0 &&
-                toNativeTokenAmount < type(uint96).max &&
-                normAmounts.transferAmount > normAmounts.toNative + normAmounts.relayerFee
-            );
-        }
+        // make some assumptions about the fuzz test values
+        vm.assume(
+            normAmounts.transferAmount > 0 &&
+            amount < type(uint96).max
+        );
+        vm.assume(
+            normAmounts.toNative > 0 &&
+            toNativeTokenAmount < type(uint96).max &&
+            normAmounts.transferAmount > normAmounts.toNative + normAmounts.relayerFee
+        );
```

Comment:
Same as previously. You can likely wrap some of these conditionals with [bound](https://book.getfoundry.sh/reference/forge-std/bound) for running the tests faster.